cob_bringup/env.sh

```diff
@@ -13,4 +13,6 @@ else
     source /opt/ros/indigo/setup.bash
 fi
 
+export ROS_IP=`hostname -I | awk '{print $1}'`
```

Comment:
what content is in `$1`?
I would like to add this to my the `.bashrc` on my laptop :wink: